### PR TITLE
Report issue modal

### PR DESCRIPTION
**Pull request by biyeun**
http://manage.dimagi.com/default.asp?231380

only resets the report issue form once after closing the modal after submission
helps prevent accidental click aways clearing the form

@benrudolph @orangejenny 
fyi @sheelio

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/js-custom/hqhandler.form.bugReport.js [hunk 1]
```diff
@@ -1,102 +0,0 @@
-$.fn.bootstrapButton = $.fn.button;
```
Comment (benrudolph) [inline corehq/apps/hqwebapp/static/hqwebapp/js-custom/hqhandler.form.bugReport.js]: 😂

File: corehq/apps/style/static/style/js/hq-bug-report.js [hunk 2]
```diff
@@ -79,6 +80,9 @@ $(function () {
     function hqwebappBugReportSucccess(data) {
         isBugReportSubmitting = false;
         $hqwebappBugReportForm.find("button[type='submit']").bootstrapButton('success').removeClass('btn-primary btn-danger').addClass('btn-success');
+        $hqwebappBugReportModal.one('hidden.bs.modal', function () {
```
Comment (benrudolph) [inline corehq/apps/style/static/style/js/hq-bug-report.js]: nice

Comment (benrudolph) [inline corehq/apps/style/static/style/js/hq-bug-report.js]: nit: could just be '$hqwebappBugReportModal.one('hidden.bs.modal', resetForm)' though i guess that means you get args passed in

**Comment by benrudolph** [comment]
👍 on ✔️

**Comment by biyeun** [comment]
@benrudolph bump

**Comment by biyeun** [comment]
also cc @orangejenny

**Pull request merged**